app/react/package.json

```diff
@@ -86,6 +86,18 @@
     "@babel/core": {
       "optional": true
     },
+    "@storybook/builder-webpack4": {
```

Comment:
Why do we need to add these?

Reply:
Because there is `@storybook/core` as dep and it has `@storybook/builder-webpack4` as optional peer dep itself, so we need to add it to satisfy transitivity. Maël did a post about it some time ago: https://dev.to/arcanis/implicit-transitive-peer-dependencies-ed0